src/ck-core/ckrdma.C

```diff
@@ -1321,6 +1347,14 @@ void handleBcastReverseEntryMethodApiCompletion(NcpyOperationInfo *info) {
     // Invoke the remote ackhandler function
     invokeRemoteNcpyAckHandler(info->destPe, info->refPtr, ncpyHandlerIdx::EM_ACK);
   }
+#if CMK_REG_REQUIRED
+  // De-register source for reverse operations when regMode = UNREG and deregeMode = DEREG
+  if(info->isSrcRegistered == 1 && info->srcRegMode == CK_BUFFER_UNREG && info->srcDeregMode == CK_BUFFER_DEREG) {
+    CmiDeregisterMem(info->srcPtr, info->srcLayerInfo + CmiGetRdmaCommonInfoSize(), info->srcPe, info->srcRegMode);
```

Comment:
Couldnt this call `deregisterSrcBuffer` instead?

Reply:
Good point, Done.